bin/MadGraph5_aMCatNLO/runcmsgrid_NLO.sh

```diff
@@ -104,8 +104,8 @@ if [ ! -e $LHEWORKDIR/header_for_madspin.txt ]; then
 #else handle external tarball
 else
     cd $LHEWORKDIR/external_tarball
-
-    ./runcmsgrid.sh $nevtjob $rnum $ncpu
```

Comment:
btw, it looks to me that if you remove nevtjob here, it is not used anywhere else, so I would suggest to remove it everywhere (cfr L71)

Reply:
I may be wrong, but I'm afraid it is used for pure MG5_aMC@NLO gridpacks. That's the purpouse of that variable.

Reply:
if I click on "view" I can't see this variable used anywhere, unless I am missing something
https://github.com/jfernan2/genproductions/blob/7eb379735966dcce8786bb43c8171fb6b7bb8a9d/bin/MadGraph5_aMCatNLO/runcmsgrid_NLO.sh

Reply:
It is stored on runscript.dat and then used in L96